Shared/sdk/SharedUtil.Hash.hpp

```diff
@@ -772,7 +772,7 @@ namespace SharedUtil
         delete[] strbuf;
     }
 
-    void TeaDecode(const SString& str, const SString& key, SString* out)
+    void TeaDecode(const SString& str, const SString& key, SString* out, const bool trimPaddingAtEnd)
```

Comment:
Is this necessary? The only place where this is false is in `Shared/sdk/SharedUtil.Tests.hpp`, so it seems like this is unnecessary.

Reply:
No, it's not.